website/homepage/src/docs/obj_doc_template.html

```diff
@@ -48,6 +48,9 @@
   {% if is_component %}
   <p class="mb-2 text-lg text-gray-500">As input, {{ obj["tags"]["preprocessing"] }}</p>
   <p class="mb-2 text-lg text-gray-500">As output, {{ obj["tags"]["postprocessing"] }}</p>
+  {% if "examples-format" in obj["tags"]  %}
+  <p class="text-lg text-gray-500">Format expected for examples: <em>{{ obj["tags"]["examples-format"] }}</em></p>
```

Comment:
Keep non-italicized for consistency with the lines above?
```suggestion
  <p class="text-lg text-gray-500">Format expected for examples: {{ obj["tags"]["examples-format"] }}</p>
```

Reply:
Done! Good catch